bus_pico_pio.go

```diff
@@ -1,9 +1,10 @@
-//go:build !cy43nopio && rp2040
+//go:build !cy43nopio && (rp2040 || rp2350)
 
 package cyw43439
 
 import (
 	"encoding/binary"
+
```

Comment:
let's keep grouping stdlib as is? `machine` is part of stdlib in terms of TinyGo, isn't it?

Reply:
Goland plugin didn't recognise it, but I removed the new line.